packages/composables/__tests__/mocks/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './mockedCollectionList';
```

Comment:
comment related to folder name :
Jest recommends to name mocks folder with underscores https://jestjs.io/docs/manual-mocks#mocking-user-modules.
So instead of `mocks` you will have `__mocks__`. 
Here you can find examples https://jestjs.io/docs/manual-mocks#examples

Reply:
Excelent, I will correct that :)